Resources/Prototypes/Datasets/ion_storm.yml

```diff
@@ -748,13 +763,12 @@
   - ENOUGH CABBAGES
   - FIVE HUNDRED AND NINETY-NINE US DOLLARS
   - FIVE TEENAGERS WITH ATTITUDE
-  - GODDAMN FUCKING PIECE OF SHIT ASSHOLE BITCH-CHRISTING CUNT-SMUGGLING SWEARING
```

Comment:
what is this, sunday school?

Reply:
i meant to reply to the comment suggesting complete removal, whoops

Reply:
yeah, removed